tpm2/test/tpm2_test.go

```diff
@@ -997,6 +997,24 @@ func TestNVReadWrite(t *testing.T) {
 		t.Fatalf("NVWrite failed: %v", err)
 	}
 
+	// Enable write lock
+	if err := NVWriteLock(rw, HandleOwner, idx, emptyPassword); err != nil {
+		t.Fatalf("NVWriteLock failed: %v", err)
+	}
+
+	// Write the data again. Should fail now because it's write-locked.
+	err := NVWrite(rw, HandleOwner, idx, emptyPassword, data, 0)
+	switch err := err.(type) {
+	case nil:
+		t.Fatalf("NVWrite succeeded after NVWriteLock")
```

Comment:
Use `t.Fatal` here as you do not have any additional values aside from the message string.